### PR TITLE
CHANGELOG updates for 0.7.3-rc1

### PR DESCRIPTION
**Pull request by niftynei**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,44 +4,43 @@ All notable changes to this project will be documented in this file.
...
-- JSON API: `fundchannel_cancel` is extended to work before funding broadcast.
 - JSON API: The parameter `exclude` of `getroute` now also support node-id.
+- JSON API: `txprepare` now uses `outputs` as parameter other than `destination` and `satoshi`
+- JSON API: `fundchannel_cancel` is extended to work before funding broadcast.
 - JSON-API: `pay` can exclude error nodes if the failcode of `sendpay` has the NODE bit set
-- JSON API: The `plugin` command now returns on error. A timeout of 20 seconds is added to `start` and `startdir`
-            subcommands at the end of which the plugin is errored if it did not complete the handshake with `lightningd`.
+- JSON API: The `plugin` command now returns on error. A timeout of 20 seconds is added to `start` and `startdir` subcommands at the end of which the plugin is errored if it did not complete the handshake with `lightningd`.
 - JSON API: The `plugin` command does not allow to start static plugins after `lightningd` startup anymore.
+- Protocol: Now fails the connection to peers with empty responses to incomplete gossip queries
```
Comment (darosior) [inline CHANGELOG.md]: Maybe it's bikeshedding but this is the response which is incomplete in this case, not the query

Comment (darosior) [reply CHANGELOG.md]: Re-reading my initial sentence it was misleading..

Comment (niftynei) [reply CHANGELOG.md]: hmm good point. i went back and took a second look at the original PR and updated this accordingly.

**Comment by niftynei** [comment]
ACK https://github.com/ElementsProject/lightning/pull/3149/commits/7b602d4b1c0e3326af6263e555a5b77cd012f6d6

**Pull request merged**